lib/datadog/appsec/processor/context.rb

```diff
@@ -22,6 +22,7 @@ def initialize(handle, telemetry:)
           @run_mutex = Mutex.new
 
           @libddwaf_tag = "libddwaf:#{WAF::VERSION::STRING}"
+          @libddwaf_fallback_result = [:ok, WAF::Result.new(:ok, [], 0.0, false, [], [])]
```

Comment:
Unfortunately, we can't define it as a constant due to lazy-load of the component and `libddwaf` require statement in `Processor`.

Reply:
I'm going to rewrite it 😞